### PR TITLE
Revert backport of #33632: Parquet & s3 I/O changes

### PR DESCRIPTION
**Pull request by alimcmaster1**
Ref: https://github.com/pandas-dev/pandas/issues/34626#issuecomment-640225867

Revert backport of https://github.com/pandas-dev/pandas/pull/33632
Backport: https://github.com/pandas-dev/pandas/pull/34173

cc @jorisvandenbossche @simonjayhawkins

### CODE REVIEW

File: pandas/tests/io/test_parquet.py [hunk 1]
```diff
@@ -489,37 +485,15 @@ def test_categorical(self, pa):
             expected = df.astype(object)
             check_round_trip(df, pa, expected=expected)
 
+    # GH#33077 2020-03-27
+    @pytest.mark.xfail(
+        locale.getlocale()[0] == "zh_CN",
```
Comment (alimcmaster1) [inline pandas/tests/io/test_parquet.py:490]: ``dom, 07 giu 2020 15:58:39 GMT``` https://dev.azure.com/pandas-dev/pandas/_build/results?buildId=36889&view=logs&j=98c84d08-34d8-513e-80be-2c581992dd5a&t=f75024c1-aff0-57fb-4f0e-35923b654e09

Seems to also fail now for `it_IT` locale ?

Comment (simonjayhawkins) [reply pandas/tests/io/test_parquet.py:490]: test_s3_roundtrip failures were why the PRs were backported as this issue (with it_IT) was not been seen on master. see OP in https://github.com/pandas-dev/pandas/pull/34173#issue-417892819

I _think_ you can just skip this, see  https://github.com/pandas-dev/pandas/pull/34173#issuecomment-628573908

Comment (alimcmaster1) [reply pandas/tests/io/test_parquet.py]: Thanks done - looks like the Doc build is failing which I think is unrelated.

**Comment by jorisvandenbossche** [comment]
The doc failure is related to https://github.com/pandas-dev/pandas/pull/34458, which needs to be backported

**Review by jorisvandenbossche — APPROVED** [review]

**Pull request merged**